src/helpers/enumStringValues.ts

```diff
@@ -0,0 +1,4 @@
+export function enumStringValues(enumerable: any) {
```

Comment:
Could this be 
```ts
function enumStringValues(enumerable: Record<string, string | number>): string[] {
    return Object.entries(enumerable).map(([, val]) => val).filter((val) => typeof val === 'string')
}
```

Reply:
blah

Reply:
I copied this off stackoverflow. How dare you change its sacred nature.